### ISSUE TITLE
ember-electron doesn't work with ember-cli-typescript

### ISSUE THREAD

**Issue by gossi** [felixrieseberg/ember-electron#246]
I tried to use typescript with `ember-cli-typescript`. After installing that package and running `ember electron`  I got the following error:

```
$ ember electron
⠧ Building2017-04-30 01:28 ember[2206] (FSEvents.framework) FSEventStreamStart: register_with_server: ERROR: f2d_register_rpc() => (null) (-22)
2017-04-30 01:28 ember[2206] (FSEvents.framework) FSEventStreamStart: register_with_server: ERROR: f2d_register_rpc() => (null) (-22)
2017-04-30 01:28 ember[2206] (FSEvents.framework) FSEventStreamStart: register_with_server: ERROR: f2d_register_rpc() => (null) (-22)
2017-04-30 01:28 ember[2206] (FSEvents.framework) FSEventStreamStart: register_with_server: ERROR: f2d_register_rpc() => (null) (-22)
events.js:163
      throw er; // Unhandled 'error' event
      ^

Error: Error watching file for changes: EMFILE
    at exports._errnoException (util.js:1050:11)
    at FSEvent.FSWatcher._handle.onchange (fs.js:1376:11)
```

**Comment by chriskrycho** [comment]
For reference, this will (hopefully, eventually, 😬) get fixed at https://github.com/emberwatch/ember-cli-typescript/issues/13, which the aforelinked issue duplicated. I'll do my best to update this issue when that one closes.

**Comment by pichfl** [comment]
Given that this is not limited to Ember-Electron but originates in the mentioned addon, I'll close this issue here.

**Comment by chriskrycho** [comment]
For viewers following along at home: this issue was fixed in https://github.com/emberwatch/ember-cli-typescript/pull/30 and that fix is in the ember-cli-typescript 0.4.0 release going out momentarily!
